aiohttp/http_writer.py

```diff
@@ -56,13 +59,18 @@ def _write(self, chunk):
             raise asyncio.CancelledError('Cannot write to closing transport')
         self._transport.write(chunk)
 
-    def write(self, chunk, *, drain=True, LIMIT=64*1024):
+    def write(self, chunk, *, drain=True, LIMIT=64 * 1024):
```

Comment:
Don't touch if you don't change the logic.
If you want update code style -- do it in a separate PR.
For this particular line I pretty happy with status quo.

Reply:
That's wasn't me, it's my emacs ;) I will revert this change

Reply:
Fixed

Reply:
Also I fixed `setup.cfg` so that this line is not altered by anyone else who uses `autopep8`

Reply:
Cool